README.md

```diff
@@ -1127,6 +1127,53 @@ flags to Lua.
 * `"TX"`: n/a.
 * `"REDIRECT"`: n/a.
 
+### xtable
+
+The `xtable` library provides support for developing netfilter xtable extensions.
```

Comment:
It would be good to add a link to a xtable reference

Reply:
this doesn't look resolved.. please, see xdp section as an example.. it links to XDP documentation.. it's a good thing especially for who is not that familiar with kernel internals..